### PR TITLE
Analyze performance of MongoDB vs. PostgreSQL

### PR DESCRIPTION
**Pull request by samliebow**
These commits are mostly utility files for database work: generating schemas, populating the databases, and then timing how long querying a bunch of records takes. I have lots of performance notes stored elsewhere, but the gist is that Postgres is faster because it can do the necessary work in one query.

I don't think there's a ton to review here, but can one of you take a look at this when you have time?

### CODE REVIEW

File: db/makeMongoData.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      count++;
+    }
+  }
+  return Object.keys(backers).map(el => +el);
+};
+
+const writeProjects = () => {
+  let notFull = true;
+  while (j < 1e7 && notFull) {
+    notFull = projects.write(JSON.stringify({
```
Comment (2000prcs) [inline db/makeMongoData.js:20]: What does 1e7 do here?

File: db/mongoTest.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  await mongoose.connect('mongodb://127.0.0.1/quickstarter');
+  const start = new Date().getTime();
+  while (countdown--) {
+    const id = Math.floor(Math.random() * 10000000);
+    promises.push(getProjAndBackers(id));
+  }
+  const times = await Promise.all(promises);
+  const end = new Date().getTime();
+  mongoose.disconnect();
+  const averageRequestTime = times.reduce((memo, time) => memo + time) / n;
```
Comment (2000prcs) [inline db/mongoTest.js:36]: What was the reason you terminated mongo connection and restarted it every time?

File: db/makeMongoData.js (hunk 1 above)
Comment (samliebow) [reply db/makeMongoData.js:20]: 1e7 is just shorthand notation for 1 * 10^7 (10 million).

File: db/mongoTest.js (hunk 2 above)
Comment (samliebow) [reply db/mongoTest.js:36]: These are outside the loop where queries are actually performed, so I only open the connection before sending all the queries and close it after they're all done (with the `await Promise.all(promises)`).

**Review by 2000prcs — APPROVED** [review]

**Pull request merged**